frontend/src/static/js/main.js

```diff
@@ -182,6 +182,10 @@ window.app = new window.Vue({
     },
 
     updateTabAuthorship(obj) {
+      // remove hashes from zoom tab if available
+      window.removeHash('zoomSince');
+      window.removeHash('zoomUntil');
+      window.removeHash('avgCommitSize');
```

Comment:
Can I know why we have this `avgCommitSize` stored in the hash?  Shouldn't it be calculated directly from the data?

Reply:
Yes it can be calculated from the data but that will require another method that calculates that. Whereas this only require to pass the already computed `avgCommitSize` to the hash.